backend/src/main/java/de/wordsmith/backend/controller/WordController.java

```diff
@@ -0,0 +1,30 @@
+package de.wordsmith.backend.controller;
+
+import de.wordsmith.backend.model.Word;
+import de.wordsmith.backend.service.PonsApiService;
+import de.wordsmith.backend.service.WordflowService;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/api/word")
+public class WordController {
+
+    private final WordflowService wordflowService;
+    private final PonsApiService ponsApiService;
+
+    public WordController(WordflowService wordflowService, PonsApiService ponsApiService) {
+        this.wordflowService = wordflowService;
+        this.ponsApiService = ponsApiService;
+    }
+
+    @GetMapping()
+    public boolean doesWordExist (@RequestParam String wordText) {
+        return ponsApiService.doesWordExist(wordText);
```

Comment:
within domain driven programming using a layer modell I would not expect to see an API class within a controller class, i.e. I would expect you to have an API service class as well as a business logic layer e.g. `ponsWordService`

Reply:
I do have a WordflowService class, makes sense to first call the method from there. Changed it